repos/spack_repo/builtin/packages/amdlibflame/package.py

```diff
@@ -83,7 +84,7 @@ class Amdlibflame(CMakePackage, LibflameBase):
     # Required dependencies
     with when("build_system=cmake"):
         generator("make")
-        depends_on("cmake@3.22:", type="build")
+        depends_on("cmake@3.26:", type="build")
```

Comment:
This needs to be added with a `when` clause, and the previous cmake minimum version should remain for older version.